web/src/assets/_site.scss

```diff
@@ -1,20 +1,19 @@
 
-
```

Comment:
This entire file is just formatting change, right?

Reply:
yes